Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Is this file needed? Else we can add it to the `.gitignore`

Reply:
I didn't notice these - IDK what created them ;)